### DISCUSSION TITLE
Laravel's `dimensions` validation rule does not work

### DISCUSSION THREAD

**Discussion by jordanhavard** [livewire/livewire#6086] [7. Bugs (archived)]
### Discussed in https://github.com/livewire/livewire/discussions/5229

<div type='discussions-op-text'>

<sup>Originally posted by **gdebrauwer** September  4, 2022</sup>
Laravel's `dimensions` validation rule does not work on Livewire's `TemporaryUploadedFile`. This is caused by the fact that the `getRealPath` method returns the file path instead of the file URL.</div>

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Reply by jordanhavard** [reply]
Thanks @joshhanley, my appologies as I must of missed this 🙏 . I'll be sure to check out the link. 
My intentions were to help auto close #5229 when those PRs were merged

**Reply by joshhanley** [reply]
@jordanhavard No worries! I tagged a v2 version this morning with a fix, is the dimensions rule still not working or is it just to get it up to standard with Laravel naming?

**Reply by jordanhavard** [reply]
Tagging for visability @joshhanley, Taylor changed the method name before merging so `imageSize` actually overrides nothing as the base method is now dimensions. 
PR #6083 and #6084 resolve this 🙏

**Comment by jordanhavard** [comment]
Duplicate of #5229
